src/lints/trait_removed_supertrait.ron

```diff
@@ -0,0 +1,62 @@
+SemverQuery(
+    id: "trait_removed_supertrait",
+    human_readable_name: "supertrait removed or renamed",
+    description: "It is a breaking change to loosen generic bounds on a type since this can break users expecting the tightened bounds.",
```

Comment:
Do you mind if we refine it together? It's a bit hard to be technically precise and simple in a few words. Trying to address the first part of your comment. We are talking about a very specific part of the "generic bounds", so:
"It is a breaking change to remove supertrait on a trait since this can break users expecting trait to also behave like the supertrait."

^ This is my OOP brain trying to translate the supertrait as inherence. Maybe there is a better way to put it?

About why it breaks, I'm not sure how to explain it without showing an example.  I tried to express that with the "users expecting trait to also behave like the supertrait" but not sure if it's enough.

Reply:
Happy to help! I have a ton of stuff going on today so I probably won't be able to circle back to this until tomorrow, just wanted to let you know so you don't worry about it :)

Reply:
Ah, thanks! That looks perfect and simple!